website/content/sdk/guides/intros/stocks/fundamental-analysis.md

```diff
@@ -1,131 +1,243 @@
-| Gross profit                                |   3.84936e+08 |     1.98969e+08 |      1.65231e+08 |  7.6712e+07  |
-| Selling general and administrative          |   4.07964e+08 |     2.52133e+08 |      1.03962e+08 |  5.9148e+07  |
-| Total operating expenses                    |   4.90874e+08 |     3.01574e+08 |      1.22461e+08 |  6.597e+07   |
-| Operating income or loss                    |  -1.05938e+08 |    -1.02605e+08 |      4.277e+07   |  1.0742e+07  |
-| Interest expense                            |   5.7796e+07  |     5.1291e+07  |      3.4002e+07  |  1.0163e+07  |
-| Total other income/expenses net             |   3.8987e+07  |     3.7169e+07  |     -7.9501e+07  | -2.0678e+07  |
-| Income before tax                           |  -1.2474e+08  |    -1.16723e+08 |     -7.0413e+07  | -1.978e+07   |
-| Income tax expense                          |   1.417e+06   | 14000           | 651000           |  0           |
-| Income from continuing operations           |  -1.26157e+08 |    -1.16737e+08 |     -7.1064e+07  | -1.978e+07   |
-| Net income                                  |  -5.824e+07   |    -1.802e+07   |     -7.1064e+07  | -1.978e+07   |
-| Net income available to common shareholders |  -5.824e+07   |    -1.802e+07   |     -7.1064e+07  | -1.978e+07   |
-| Basic EPS                                   | nan           |    -0.11        |     -0.43        | -0.12        |
-| Diluted EPS                                 | nan           |    -0.28        |     -0.43        | -0.12        |
-| Basic average shares                        | nan           |     1.70507e+08 |      1.66243e+08 |  1.66243e+08 |
-| Diluted average shares                      | nan           |     4.75697e+08 |      1.66243e+08 |  1.66243e+08 |
-| EBITDA                                      | nan           |    -1.5991e+07  |     -1.7912e+07  | -2.795e+06   |
+| Breakdown                                  |   2022-08-31 |   2021-08-31 |   2020-08-31 |   2019-08-31 |
+|:-------------------------------------------|-------------:|-------------:|-------------:|-------------:|
+| Cash and cash equivalents                  |   -0.0937111 |   -0.0830007 |   0.464337   |            0 |
+| Other short-term investments               |   -0.0774264 |   -0.107977  |  -0.0301887  |            0 |
+| Total cash                                 |   -0.0924846 |   -0.0849305 |   0.408831   |            0 |
+| Net receivables                            |    0.242928  |    0.163226  |   0.00977199 |            0 |
+| Inventory                                  |    0.259726  |    0.161166  |   0.0743308  |            0 |
+| Other current assets                       |    0.14253   |    0.282502  |  -0.0792079  |            0 |
+| Total current assets                       |    0.108151  |    0.0492532 |   0.19736    |            0 |
+...continued
+
+Cashflow and Income statements function in the same way as `balance` does; `cash` and `income` respectively.
```

Comment:
maybe just put as examples here instead of `cash` and `income`

Cashflow and Income statements function in the same way as as the previous command; `openbb.stocks.fa.cash` and `openbb.stocks.fa.income` respectively.

Reply:
done